components/integration/production/kustomization.yaml

```diff
@@ -1,14 +1,13 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1 kind: Kustomization
```

Comment:
This doesn't seem like a valid change,, the `kind: Kustomization` should be in a new line.